cluster/gce/util.sh

```diff
@@ -2113,6 +2120,9 @@ function kube-up() {
     create-network
     create-subnetworks
     detect-subnetworks
+    if [[ ${GCE_PRIVATE_CLUSTER:-} == "true" ]]; then
```

Comment:
For consistency with delete-cloud-nat-router, can you please move this if to create-cloud-nat-router function?

Reply:
Sure, done.